### DISCUSSION TITLE
Welcome to MiKTeX Discussions!

### DISCUSSION THREAD

**Discussion by edocevoli** [MiKTeX/miktex#670] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by johnhdavies** [comment]
I have used LaTeX for many years but have more recently started using it on Windows with MiKTeX.

**Comment by saisth02** [comment]
I've thi message when using Rstudio
Error running C:/Users/Oussa/AppData/Local/Programs/MiKTeX/miktex/bin/x64/pdflatex.exe (exit code -1073740791)
I need help! Thank's

**Reply by creviherol** [reply]
I have a similar problem

**Comment by Abdellah35** [comment]
Is Mik-Tex v22 supported by Windows server 2012?
If not please let me know the supported version.
Thank you! for your support.

**Comment by Athene-ai** [comment]
Hi!

nice to meet you all :-)

**Comment by Adelab5** [comment]
Please am seeing thus error in my latex "MiKTeX could not find the script engine 'perl' which is required to execute 'makeglossaries'. "

**Comment by abdulloh031997** [comment]
code="28", url="https://ctan.yazd.ac.ir/systems/win32/miktex/tm/packages/miktex-zzdb3-2.9.tar.lzma" error timeOut

**Comment by aboudsawan** [comment]
Hello, 
Tried to install MikTex on Windows, but I got the following error "ctan.yazd.ac.ir took too long to respond"
So I searched for the installation file "basic-miktex-24.1-x64.exe" on Google and found another source to install MikTex.

Now when I do update, I get  the following:
Timeout was reached

Detail:
code="28", url="https://ctan.yazd.ac.ir/systems/win32/miktex/tm/packages/miktex-zzdb3-2.9.tar.lzma".

please provide help on this issue... why am I being directed to ""ctan.yazd.ac.ir" for installation or update?

Thnx

ps: geographically, I am close to Iran, do I need to do something manually to alter this installation/update setup?

**Reply by lamaplays** [reply]
hi did u fine a solution?

**Comment by izqilar** [comment]
I had a problem installing MiKTeX, and an error occurred while the installation was almost complete:

An error occurred:
  source file: 
  source line: 0
  message: The executed process did not succeed.
  info: fileName="d:\miktex-portable\texmfs\install\miktex\bin\x64\mpm.exe", exitCode="1"

**Comment by amna53123** [comment]
I have to google it since the download link in mikttex.org is not working. I downloaded the MiKTeX Setup Utility "miktexsetup-5.5.0+1763023-x64.zip" you can find it here https://ctan.org/tex-archive/systems/win32/miktex/setup/windows-x64?lang=en
